cogs/utility.py

```
@@ -0,0 +1,410 @@
+import discord
+from discord.ext import commands
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)